### DISCUSSION TITLE
Voucher behaviour for dummies (me)

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#9867] [Deployment]
Hi everyone!

I'd like to ask few things about vouchers behaviour, I'm sorry I can't grasp what's going on 
under the hood.

So let's say I have this situation:
I've created a simple voucher of fixed amount, with a minimum value to be available, 
active on my channels, applied on specific products BUT only once per order.
I've attached few images of the situation just to make it clearer I hope.

![voucher1](https://user-images.githubusercontent.com/31857092/170758646-89635069-96a5-41bd-af1c-62117c190df1.jpg)

![voucher2](https://user-images.githubusercontent.com/31857092/170758758-8a1c0a47-209d-40eb-aa61-4008ad9d1934.jpg)

![voucher3](https://user-images.githubusercontent.com/31857092/170758874-51628270-60e6-47ec-ad32-9a154a04cf0b.jpg)

The fact is:
looking at many orders and purchases, I've witnessed different behaviours, that I
summarize briefly:
- voucher entirely applied to a single cart item(s)
- voucher amount splitted between different items
- voucher applied asymmetrically between two or more items: let's say on a voucher amount of 70$, 50$ on an item and 20$ on the other.

**_So I'm actually in this position where I've to talk to stakeholders explaining exactly how it works, and I have to be honest
I feel myself a little bit like Kermit against Ironman._**

I know you're patient and good-willing people, I kindly count on you for this explanation if possible.

Best regards,
Lorenzo

**Comment by Anubiella** [comment]
@DidoMarchet you know what I mean buddy

**Comment by carlosa54** [comment]
I reported a bug in https://github.com/saleor/saleor/issues/8910 some time ago that I don't know if it relates to what you're describing but I think it's related to the same logic

**Comment by timuric** [comment]
@Anubiella Please disable "Apply once per order". It is a misleading label which means "Apply to a single item"
